university-of-gothenburg-apa-swedish-legislations.csl

```diff
@@ -129,7 +129,7 @@
       <if type="thesis report" match="any">
         <choose>
           <if variable="DOI" match="any">
-            <text variable="DOI" prefix="doi: "/>
+            <text variable="DOI" prefix="https://dx.doi.org/"/>
```

Comment:
@bjornekstrom, do you know if https://doi.org/ would also be acceptable? "https://dx.doi.org/" as a prefix is actually already outdated, and the shorter https://doi.org/ is now the officially recommended format. See https://www.zotero.org/support/kb/doi_in_apa for details.

Reply:
@rmzelle Yes, this works better. Thanks for your reply. I have made the necessary amendments now.